backbone/src/main/java/org/researchstack/backbone/task/factory/WalkingTaskFactory.java

```diff
@@ -142,7 +143,7 @@ public static OrderedTask shortWalkTask(
                     recorderConfigList.add(new AccelerometerRecorderConfig(AccelerometerRecorderIdentifier, sensorFreq));
                 }
                 if (!optionList.contains(TaskExcludeOption.DEVICE_MOTION)) {
-                    recorderConfigList.add(new AccelerometerRecorderConfig(DeviceMotionRecorderIdentifier, sensorFreq));
+                    recorderConfigList.add(new DeviceMotionRecorderConfig(DeviceMotionRecorderIdentifier, sensorFreq));
```

Comment:
Maybe there's a way we could refactor the recorderConfig and exclusion logic to reduce the repeats